nimble/controller/src/ble_ll.c

```diff
@@ -1277,6 +1280,46 @@ ble_ll_read_supp_features(void)
     return g_ble_ll_data.ll_supp_features;
 }
 
+/**
+ * Sets the features controlled by the host.
+ *
+ * @return HCI command status
+ */
+int
+ble_ll_set_host_feat(const uint8_t *cmdbuf, uint8_t len)
+{
+#if MYNEWT_VAL(BLE_VERSION) >= 52
+    const struct ble_hci_le_set_host_feat_cp *cmd = (const void *) cmdbuf;
```

Comment:
you also need to check if len == sizeof(*cmd) here

Reply:
thx